aw/Extensions/Reach.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  Reach.swift
+//  aw
+//
+//  Created by Alex Kerney on 3/24/18.
+//  Copyright © 2018 Alex Kerney. All rights reserved.
+//
+
+import Foundation
+import MapKit
+
+extension Reach {
+
+    var readingFormatted: String {
+        guard let lastGuageReading = lastGageReading, let unit = unit else {
```

Comment:
Misspelled. I'd also recommend committing to spelling it as either "gage" or "gauge".  I chose "gage" on Android because it's shorter.